shmem.c

```diff
@@ -12,7 +12,7 @@
 #include "shmem.h"
 
 /// The version of shared memory used
-#define SHARED_MEMORY_VERSION 2
+#define SHARED_MEMORY_VERSION 3
```

Comment:
This will now need to be version 4, due to version 3 being taken by the overTime rewrite.

Reply:
Due to the version jump that will be included in FTL v4.2.3, this now needs to be `5`.